README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
+
+- Make it easy to build and package on all supported platforms.
+- Provide abstractions to avoid platform specific quirks.
```

Comment:
```suggestion
- Provide a flexible and ergonomic API that abstracts common platform behaviour.
- Make it possible to interact with the platform directly when more control is necessary.
```

Reply:
Just to note that *abstracts common platform behaviour* wouldn't cover things like the multi-click api in #859 because that's not a common platform behavior. It's an abstraction that builds the behavior.